lib/desired-caps.js

```diff
@@ -0,0 +1,14 @@
+let geckodriverConstraints = {
+  geckodriverPort: {
+    isNumber: true
+  },
+  // duplicate of above with better spelling
+  geckoDriverPort: {
+    isNumber: true
+  },
+  firefoxInitialUrl: {
```

Comment:
I don't know this capability but would it make sense to use [`firefox-profile`](https://www.npmjs.com/package/firefox-profile) to make a much larger set of options accessible?

Reply:
yep, we have the same strategy for the chromedriver. Some of the most recently used options are moved to the "top" level, like the initial url one, but a user has a possibility to always provide a full set of driver options as a separate object. See https://github.com/appium/appium-android-driver/blob/4aad401eb3ff86fe87e4dd915d275c3a478c2f7b/lib/commands/context.js#L372

Reply:
Removed